src/themes/masschallenge/elements/button.overrides

```diff
@@ -4,3 +4,11 @@
 .ui.step.mini.button {
   min-width: 124px;
 }
+
+.ui.whitebg.button {
+  background: #ffffff !important;
+}
```

Comment:
I do not think we should use `!important` in this case. Recoloring the button should not come at such a cost, I'll adjust the UI to work better within our current framework. Please remove.